src/app.py

```diff
@@ -14,7 +14,7 @@
 server = app.server
 
 #Read the CSV file
-df = pd.read_csv('data/raw/van_weather_1974-01-01_2024-03-15.csv', encoding='latin-1', index_col='date', parse_dates=True)
+df = pd.read_csv('/Users/katherinechen/Desktop/532_lab/DSCI-532_2024_10_vanweather/data/raw/van_weather_1974-01-01_2024-03-15.csv', encoding='latin-1', index_col='date', parse_dates=True)
```

Comment:
```suggestion
df = pd.read_csv('data/raw/van_weather_1974-01-01_2024-03-15.csv', encoding='latin-1', index_col='date', parse_dates=True)
```

Let us use this relative path instead. Just make sure when you run the code, you run from root. And use python src/app.py